crates/bevy_pbr/src/render/pbr_functions.wgsl

```diff
@@ -196,38 +196,35 @@ fn pbr(
     // point lights
     for (var i: u32 = offset_and_counts[0]; i < offset_and_counts[0] + offset_and_counts[1]; i = i + 1u) {
         let light_id = get_light_id(i);
-        let light = point_lights.data[light_id];
         var shadow: f32 = 1.0;
         if ((mesh.flags & MESH_FLAGS_SHADOW_RECEIVER_BIT) != 0u
-                && (light.flags & POINT_LIGHT_FLAGS_SHADOWS_ENABLED_BIT) != 0u) {
+                && (point_lights.data[light_id].flags & POINT_LIGHT_FLAGS_SHADOWS_ENABLED_BIT) != 0u) {
```

Comment:
could you use a `&light` here like you did in other functions?

Reply:
Yes, but it was only one access so I instead just refer to it directly.

Reply:
Ah, right, I saw another one below, but I didn't realize it was a different loop, makes sense then